extensions/sql-bindings/src/sqlbindings.d.ts

```diff
@@ -0,0 +1,97 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+
+declare module 'sqlbindings' {
```

Comment:
Let's name this sql-bindings instead to match the name format of the extension (update the filename too)